docs/details/shards/GFX/Drawable.md

```diff
@@ -0,0 +1,47 @@
+The drawable shard is used to define an instance of a drawn object, it usually has a transform (in world space) and a mesh to draw.
+
+--8<-- "includes/drawable_refactor.md"
+
+## Params
+
+The `Params` parameter contains a table of values that will be passed to the shader when this drawable is rendered.
+
+!!! info Defining shader parameters
+    See [GFX.Feature](../Feature) about where shader parameters are defined and used
+
+## Constant/Dynamic
+
+Shader parameters and the drawable transform can be set as either constant parameters or dynamic ones.
+
+### Constant parameters
+
+Constant parameters are passed through the input table. They are read and stay the same until this function is called again.
+
+```clojure
+{:Mesh .mesh :Transform .transform} (GFX.Drawable) >= .my-drawable
+```
+
+### Dynamic parameters
+
+Dynamic parameters are passed as parameters to the shard, the same fields can be set, except for the mesh. They are referenced and when their value changes will reflect this change in the rendered result.
+
+```clojure
+.. >= .dynamic-color
+.. >= .dynamic-transform
+{:Mesh .mesh} (Drawable :Transform .dynamic-transform :Params {:baseColor .dynamic-color}) >= .my-drawable
+```
+
+## Intended usage
+
+The currently intended way to set up drawables without frequent changes is to set them up from within a setup block. Any dynamic parameters that might change should then be passed as a parameter to the shard.
```

Comment:
```suggestion
To avoid frequent changes, you should set up drawables in a `Setup` block and pass their dynamic parameters to the shard.
```
Might need to reword this too as the definition is a little unclear.

Reply:
Reworded